app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for peoples in people:
+        Person(peoples["name"], peoples["age"])
+    for peoples in people:
+        result_people = Person.people[peoples["name"]]
+        if "wife" in peoples and peoples["wife"] is not None:
+            result_people.wife = Person.people[peoples["wife"]]
+
+        if "husband" in peoples and peoples["husband"] is not None:
+            result_people.husband = Person.people[peoples["husband"]]
```

Comment:
Similar to line 16, ensure that the `husband` key corresponds to a valid name in the `Person.people` dictionary to avoid `KeyError` exceptions.